src/components/textFields/LargeTextField.js

```diff
@@ -53,7 +53,7 @@ const LargeTextField = (props) => {
          }
      }
 
-
+      /* Calls when text is updated, evaluates the validity of text based on TextFieldValidation component */
       const onTextChange = (text) => {
```

Comment:
I would definitely add more comments to the body of this function explaining how the validation works.